config/database.yml

```diff
@@ -17,9 +17,12 @@
 default: &default
```

Comment:
Now that we have `database.yml.sample` this file can be removed from version control

Reply:
The file has been added to gitignore and removed.